### ISSUE TITLE
Signed In returns No results

### ISSUE THREAD

**Issue by ChaseWPDEV** [patrickkfkan/volumio-ytmusic#6]
Using plugin version 1.1.2 or 1.1.3. After signing in to YouTube Music, attempting to go to Home->YouTube Music->{any of the 4 tabs: Home, Explore, Library, History} a red "Error no results" red warning comes up.

Able to play as non-logged in user.

**Comment by ChaseWPDEV** [comment]
Logs:
```
ov 19 11:45:18 music volumio[3196]: info: CoreCommandRouter::executeOnPlugin: ytmusic , handleBrowseUri 
Nov 19 11:45:18 music volumio[3196]: info: [ytmusic-browse] browseUri: ytmusic/generic@endpoint:o=%7B%22type%22%3A%22browse%22%2C%22payload%22%3A%7B%22browseId%22%3A%22FEmusic_home%22%7D%7D 
Nov 19 11:45:19 music volumio[3196]: error: Request to https://www.youtube.com/youtubei/v1/browse?prettyPrint=false&alt=json failed with status 400 Error: Request to https://www.youtube.com/youtubei/v1/browse?prettyPrint=false&alt=json failed with status 400 
Nov 19 11:45:19 music volumio[3196]:     at HTTPClient.<anonymous> (/data/plugins/music_service/ytmusic/node_modules/volumio-youtubei.js/bundle/node.cjs:9501:13) 
Nov 19 11:45:19 music volumio[3196]:     at Generator.next (<anonymous>) 
Nov 19 11:45:19 music volumio[3196]:     at fulfilled (/data/plugins/music_service/ytmusic/node_modules/volumio-youtubei.js/bundle/node.cjs:215:30) 
Nov 19 11:45:19 music volumio[3196]:     at runMicrotasks (<anonymous>) 
Nov 19 11:45:19 music volumio[3196]:     at processTicksAndRejections (internal/process/task_queues.js:93:5) 
Nov 19 11:45:19 music volumio[3196]: error: Failed to execute browseSource: Error: Request to https://www.youtube.com/youtubei/v1/browse?prettyPrint=false&alt=json failed with status 400 
```

**Comment by patrickkfkan** [comment]
Fixed in v2.1.0 for Bookworm-based Volumio
